libraries/DateTime/test/dateTimeLibrary.js

```diff
@@ -0,0 +1,359 @@
+//@ts-nocheck
+const fs = require('fs-extra')
+const path = require('path')
+const Deployer = require('forgae-lib').Deployer
+const contractDirectory = `${process.cwd()}/contracts/`;
+const exampleContract = 'ExampleContract.aes'
+const contract = fs.readFileSync(path.resolve(`${contractDirectory}/${exampleContract}`), 'utf-8')
+const actualContractName = 'ExampleWithLibrary.aes'
+const actualContract = contractDirectory + actualContractName;
+const {
+  createActualContract,
+  removeFiles
+} = require('./utils/utils')
+
+describe('DateTime Library', () => {
+  let deployedContract
+  let deployer
+
+  let year = 2012
+  let month = 6
+  let day = 8
+  let hour = 11
+  let minute = 58
+  let second = 59
+  const starting_timestamp = 1339156739 //2012-06-08T11:58:59
+
+  before(async () => {
+    const ownerKeyPair = wallets[0]
+
+    deployer = new Deployer('local', ownerKeyPair.secretKey)
+    createActualContract(contract, contractDirectory, actualContract)
+    deployedContract = deployer.deploy(actualContract)
+  })
+
+  it('should deploy the contract', async() => {
+    assert.isFulfilled(deployedContract, 'Could not deploy the ExampleContract Smart Contract')
+    deployedContract = await deployedContract
+  })
+
+  it('should get correct year', async () => {
```

Comment:
tests against another library or implementation e.g. new Date() or moment-js with random dates would be nice, instead of having single fixed test-cases for one date

Reply:
done